include/appserver.h

```diff
@@ -368,10 +367,20 @@ class AppServerTsx
   /// Clones the request.  This is typically used when forking a request if
   /// different request modifications are required on each fork.
   ///
+  /// NOTE: This method only exists for backwards compatibilty.
```

Comment:
This warning should be a bit stronger to suggest people shouldn't be using it. e.g. "This method is DEPRECATED and must not be used in new code."